src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutDispatcherProcessor.cs

```diff
@@ -32,7 +32,16 @@ public bool Handle(TransportMessage message)
 
             if (TimeoutsPersister.TryRemove(timeoutId, out timeoutData))
             {
-                MessageSender.Send(timeoutData.ToTransportMessage(), timeoutData.ToSendOptions(Configure.LocalAddress));
+                try
+                {
+                    MessageSender.Send(timeoutData.ToTransportMessage(), timeoutData.ToSendOptions(Configure.LocalAddress));
+                }
+                catch
+                {
+                    timeoutData.Id = null;
+                    timeoutData.Time = DateTime.UtcNow.AddSeconds(5);
+                    TimeoutsPersister.Add(timeoutData);
```

Comment:
Shouldn't we at least log that this is happening?
Also, if we fail to re-add, should we log an additional entry stating that timeout was lost?